lib/postgrex/types.ex

```diff
@@ -92,6 +93,17 @@ defmodule Postgrex.Types do
     """
   end
 
+  @doc false
+  @spec reload_query({pos_integer, non_neg_integer, non_neg_integer}, [oid, ...], state) :: binary | nil
+  def reload_query(version, oids, {_, table}) do
+    case Enum.reject(oids, &:ets.member(table, &1)) do
```

Comment:
My understanding is that on the happy path, we should eliminate all oids as already loaded, right?

If that's the case, isn't this repeated call into ets be kind of expensive?

Reply:
It should be very cheap, the `oids` here are a list of oids that the caller found were not in the table when the same process requested the bootstrap lock and these `oids` can only be from be from the list of params OR list of columns for a single query. The check is being carried out because its possible another process got the lock first and handled a subset of the oids already, so its hard to say what the happy path is here. We could use a `:ets.select` instead but its less clear and not obviously faster. 

This does raise two issues:

1) If there is an `oid` we don't know in params should we also collect oids we don't know in results?

2) There is a bug as we could have duplicate oids in this list.

Reply:
I've fixed these two issues.